### PR TITLE
Explain that Snap Store Proxy needs to be online

### PR DESCRIPTION
**Pull request by suligap**

### CODE REVIEW

File: en/index.md [hunk 1]
```diff
@@ -4,9 +4,19 @@ title: Introduction
...
+[Snap Store](https://snapcraft.io/store) for your devices. Devices are
+registered with the proxy, and all communication with the Store will flow
+through the proxy.
+
+!!! NOTE:
+    In order to serve its client devices, a Snap Store Proxy needs to be online
+    and connected to the general [Snap Store](https://snapcraft.io/store). This
+    is a requirement, even though Snap Store Proxy caches downloaded snap files,
+    which substantially reduces internet traffic. There's currently no generally
+    available offline mode for the Snap Store Proxy itself. See, the up-to-date
```
Comment (doismellburning) [inline en/index.md:17]: `See the` probably reads slightly better than `See, the`

Comment (suligap) [reply en/index.md]: Thanks!

**Comment by sparkiegeek** [comment]
Hmm this probably should call out `snap-store-proxy check-connections`, to show how the tool itself documents its own network requirements, and checks them!

**Comment by suligap** [comment]
@sparkiegeek thanks! Good point. https://github.com/canonical-ols/snapstore-snap-docs/pull/20 should fix this.

**Pull request merged**